### PR TITLE
Fix javadoc violations

### PR DESCRIPTION
**Pull request by psoujany**
This PR fixes below violation seen in openj9 docs generated using Semeru JDK8.

    1. IBMA_Color_Contrast_WCAG2AA
    Info: Violation Text contrast of 2.21 with its background is less than the WCAG AA minimum requirements for text of size  14px and weight of 700 - 1.4.3 Contrast (Minimum)

    2. WCAG20_Table_CapSummRedundant
    Info: Violation The table summary duplicates the caption - 1.3.1 Info and Relationships 

Note: We've backport for IBMA_Color_Contrast_WCAG2AA which is in a stylesheet.css which contains 4 intermediate backports containing dependent patches being a challenge. Hence, raising the PR with required changes which fixes the violations in JDK8.

### CODE REVIEW

**Comment by pshipton** [comment]
If this is a backport or is related to changes in a higher version (11, 17, etc), pls indicate what those changes are in OpenJDK.

File: langtools/src/share/classes/com/sun/tools/doclets/formats/html/markup/HtmlTree.java [hunk 1]
```diff
@@ -23,6 +23,12 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2010, 2023 All Rights Reserved
```
Comment (keithc-ca) [inline langtools/src/share/classes/com/sun/tools/doclets/formats/html/markup/HtmlTree.java:28]: The first year should be 2023.

File: langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css [hunk 2]
```diff
@@ -571,4 +587,4 @@ div.block div.block span.interfaceName {
 
 div.contentContainer ul.blockList li.blockList h2{
     padding-bottom:0px;
-}
+}
```
Comment (keithc-ca) [inline langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css:590]: Please revert this unnecessary change.

**Review by keithc-ca — CHANGES_REQUESTED** [review]

**Comment by keithc-ca** [comment]
The commit message and the description here should say "Fix ...", not "Fixing ...".

**Comment by psoujany** [comment]
@pshipton 
For IBMA_Color_Contrast_WCAG2AA below is the backport PR
https://github.com/openjdk/jdk17u-dev/commit/e2eab3c1b7d55860e705ae6f924a2a3976f76f48

For WCAG20_Table_CapSummRedundant, there's no backport PR.

File: langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css (hunk 2 above)
Comment (psoujany) [reply langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css:590]: Only line numbers are changed here right, I couldn't understand what needs to be reverted here. Please suggest, Thank you.

Comment (pshipton) [reply langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css:590]: I believe what Keith is asking is to add a newline at the end of line 590 so we don't see that difference from the original code.

**Comment by keithc-ca** [comment]
Why did you not pursue https://github.com/openjdk/jdk8u-dev/pull/114 further?

File: langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css (hunk 2 above)
Comment (psoujany) [reply langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css:590]: Thanks for the clarification.

**Comment by psoujany** [comment]
> Why did you not pursue [openjdk/jdk8u-dev#114](https://github.com/openjdk/jdk8u-dev/pull/114) further?

This backport PR has dependencies on 4 of the intermediate patches which contain many changes to the sheet, also paths of the stylesheet.css in JDK8 and JDK11 is different which is not a straight backport. After discussing with JCL team it's been decided to only port those changes which fixes the violation instead of full backport so we couldn't take that PR further.

File: langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css [hunk 3]
```diff
@@ -572,3 +588,4 @@ div.block div.block span.interfaceName {
 div.contentContainer ul.blockList li.blockList h2{
     padding-bottom:0px;
 }
+
```
Comment (pshipton) [inline langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css:591]: Now there is an extra blank line. There should be a way so the end of the file is no different from the original.

Comment (keithc-ca) [reply langtools/src/share/classes/com/sun/tools/doclets/internal/toolkit/resources/stylesheet.css:591]: @psoujany Can you fix this, please?

**Comment by keithc-ca** [comment]
Jenkins compile plinux,win32 jdk8

**Review by keithc-ca — APPROVED** [review]
